### ISSUE TITLE
GET Request Body Error in `OpenAPI` with `v0.5.0`

### ISSUE THREAD

**Issue by horizon365** [hbakri/django-ninja-crud#415]
version 0.5.0
when use in openapi, it get error.
![图片](https://github.com/hbakri/django-ninja-crud/assets/17285011/00876bf9-c9f4-48aa-b505-8d89a56c13c4)
it may effect all url support by ninja-crud.

**Comment by robinsandstrom** [comment]
Seems to be in this code segment: 

`    @functools.cached_property
    def view_func(self) -> ViewFunction:
        """
        Return the wrapped view function with path, query, and request body parameters
        annotated, and any decorators applied.

        Returns:
            ViewFunction: The wrapped view function.
        """

        def wrapped_view_function(
            request: django.http.HttpRequest,
            path_parameters: Any = ninja.Path(default=None, include_in_schema=False),
            query_parameters: Any = ninja.Query(default=None, include_in_schema=False),
            request_body: Any = ninja.Body(default=None, include_in_schema=False),
        ) -> Any:
            return self.view_function(request, path_parameters, query_parameters, request_body)

        wrapped_view_function.__annotations__.update(
            {
                "path_parameters": self.path_parameters,
                "query_parameters": self.query_parameters,
                "request_body": self.request_body,
            }
        )
        if self.view_function_name is not None:
            wrapped_view_function.__name__ = self.view_function_name

        for decorator in reversed(self.decorators):
            wrapped_view_function = decorator(wrapped_view_function)

        return wrapped_view_function
`

If I remove request_body from the argument of wrapped_view_function the error disappears. But not sure how that will work on post requests. Any PRs?

**Comment by jboutros** [comment]
Also ran into this issue - all `GET` requests in my OpenAPI config specify an optional request body which is tripping up some clients.

As a temporary workaround, I'm using the `openapi_extra` mechanism in ninja to override the request body definition. This creates a technically invalid openapi spec but solves my problem at the client level. YMMV.

Defining a view in this fashion looks something like this:
```
retrieve_profile_by_user_id = views.ReadView(
        get_model=lambda request, path_parameters: Profile.objects.get(user_id=path_parameters.id),
        path="/user/{id}",
        operation_kwargs={
            "openapi_extra": {
                "requestBody": None
            }
        }
    )
```

Ideally we'd be able to suppress the inclusion of a request body on methods such as `GET`, `DELETE`, and `HEAD` as it is [no longer allowed](https://swagger.io/docs/specification/describing-request-body/) on those methods as of [openapi 3.0](https://spec.openapis.org/oas/v3.0.3#fixed-fields-7)

**Comment by hbakri** [comment]
Hi everyone! I apologize for the delay in responding to this issue. I was traveling after my talk at DjangoCON Europe in Vigo and it took some time to get back to development. But good news, I'm finally back! 🌞

I understand the problem with the forced inclusion of request components in all CRUD endpoints, which causes the 'GET Request cannot have a body' error. I've been working on a fix and have identified a solution. You can track the progress in PR #424. The fix will be included in v0.6, expected later this week.

Thank you for your patience and contributions!
